src/index.html

```diff
@@ -2,26 +2,447 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
+
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+
     <title>Catalog</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
     <link
       rel="preconnect"
       href="https://fonts.gstatic.com"
     />
+
     <link
-      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
       rel="stylesheet"
     />
+
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="/src/styles/index.scss"
```

Comment:
The `href` attribute is pointing to a `.scss` file. Browsers cannot directly interpret SCSS files. You should compile the SCSS to CSS and link the resulting CSS file instead.